Example/SPM-AppStoreConnect-Swift-SDK/Sources/SPM-AppStoreConnect-Swift-SDK/main.swift

```diff
@@ -6,15 +6,38 @@ import AppStoreConnect_Swift_SDK
 private let configuration = APIConfiguration(issuerID: "<YOUR ISSUER ID>", privateKeyID: "<YOUR PRIVATE KEY ID>", privateKey: "<YOUR PRIVATE KEY>")
 var provider: APIProvider = APIProvider(configuration: configuration)
 
-provider.request(Endpoints.apps()) { (result) in
-    switch result {
-    case .success(let appsResponse):
-        print("Did fetch \(appsResponse.data.count) apps")
-        exit(EXIT_SUCCESS)
-    case .failure(let error):
-        print("Something went wrong fetching the apps: \(error)")
-        exit(EXIT_FAILURE)
-    }
+provider.request(.apps(
+    select: [.apps([.name]), .builds([.version, .processingState, .uploadedDate])],
+    include: [.builds],
+    sortBy: [.ascending(.bundleId)],
+    limits: [.apps(2)])) {
+        switch $0 {
+        case .success(let appsResponse):
+            guard
+                let app = appsResponse.data.first,
+                let name = app.attributes?.name,
+                let buildVersions = appsResponse.included?.compactMap({ included -> String? in
+                    if case let .build(build) = included {
+                        return build.attributes?.version
+                    }
+                    return nil
+                })
+                else {
+                    print("Could not find requested relationships!")
+                    exit(EXIT_FAILURE)
+            }
+            
+            print("App name is \(name)")
+            print(" - successfully got \(buildVersions.count) builds included")
+            for version in buildVersions {
+                print("  - \(version)")
+            }
```

Comment:
Can you also update the CocoaPods example project?

Reply:
Yes, done.